content/news/2024-09-30-maplibre-newsletter-september-2024/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "MapLibre Newsletter September 2024"
+date: "2024-09-30"
+categories: ["newsletter"]
+authors: [ramyaragupathy, ianthetechie, harel, bart]
+draft: false
+---
+
+# MapLibre Newsletter - September 2024
```

Comment:
Please check the image added

Reply:
Thanks! Can you upload it as svg instead of PNG? Images with text always look pixelated otherwise. Probably easiest is to copy-paste https://github.com/maplibre/maplibre.github.io/blob/main/content/news/2024-09-02-maplibre-newsletter-august-2024/maplibre-radar-logo.svg and replace the radar logo with the microsoft logo.